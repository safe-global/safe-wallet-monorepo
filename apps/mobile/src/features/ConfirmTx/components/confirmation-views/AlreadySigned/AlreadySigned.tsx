import React from 'react'
import { Text, YStack } from 'tamagui'
import { useSafeAreaInsets } from 'react-native-safe-area-context'

export function AlreadySigned() {
  const insets = useSafeAreaInsets()

  return (
    <YStack paddingBottom={insets.bottom ? insets.bottom : '$4'}>
      <Text fontSize="$4" fontWeight={400} textAlign="center" color="$textSecondaryLight" marginBottom="$2">
        Can be executed once the threshold is reached.
      </Text>
    </YStack>
  )
}
