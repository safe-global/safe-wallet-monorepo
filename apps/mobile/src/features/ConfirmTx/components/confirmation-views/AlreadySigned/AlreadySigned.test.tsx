import React from 'react'
import { render } from '@/src/tests/test-utils'
import { AlreadySigned } from './AlreadySigned'

describe('AlreadySigned', () => {
  it('renders correctly with all required elements', () => {
    const { getByText } = render(<AlreadySigned />)

    expect(getByText('Can be executed once the threshold is reached.')).toBeTruthy()
  })

  it('matches snapshot', () => {
    const { toJSON } = render(<AlreadySigned />)
    expect(toJSON()).toMatchSnapshot()
  })
})
