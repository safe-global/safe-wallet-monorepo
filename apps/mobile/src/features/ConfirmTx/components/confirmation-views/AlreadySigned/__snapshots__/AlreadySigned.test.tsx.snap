// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlreadySigned matches snapshot 1`] = `
<View>
  <View
    style={
      {
        "flexDirection": "column",
        "paddingBottom": 16,
      }
    }
  >
    <Text
      style={
        {
          "color": "#A1A3A7",
          "fontFamily": "DM Sans",
          "fontSize": 14,
          "marginBottom": 8,
          "textAlign": "center",
        }
      }
      suppressHighlighting={true}
    >
      Can be executed once the threshold is reached.
    </Text>
  </View>
</View>
`;
